rel/env.sh.eex

```diff
@@ -14,8 +14,9 @@
 # Set the release to work across nodes. If using the long name format like
 # the one below (my_app@127.0.0.1), you need to also uncomment the
 # RELEASE_DISTRIBUTION variable below. Must be "sname", "name" or "none".
+
 export RELEASE_DISTRIBUTION=name
 export RELEASE_NODE=<%= @release.name %>@127.0.0.1
-export DATABASE_URL="ecto://postgres:${GAPP_PASS}@${GAPP_DB}:5432/mind_the_gapp_2"
-export SECRET_KEY_BASE="${GAPP_PASS}"
-export CORS_ALLOWED_ENDPOINT="${CORS_ALLOWED_ENDPOINT}"
+export DATABASE_URL="ecto://postgres:$GAPP_PASS@$GAPP_DB:5432/mind_the_gapp_2"
```

Comment:
Just an FYI, it is conventional to pass in the port and database name as parameters as well. That would afford you the opportunity to not do a code change if the database name changes (i.e. mind_the_gapp_**2**) 😄